specification/resources/resource-manager/Microsoft.Solutions/stable/2017-09-01/managedapplications.json

```diff
@@ -869,6 +869,23 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+    "/providers/Microsoft.Solutions/operations": {
+      "get": {
+        "tags": [
+          "Features"
+        ],
+        "operationId": "Features_Operations",
```

Comment:
Incorrect. This should be Microsoft.Solutions operations API, not features. Please fix them